cmd/libs/go2idl/generator/default_package.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generator
+
+import (
+	"k8s.io/kubernetes/cmd/libs/go2idl/types"
+)
+
+// DefaultPackage contains a default implentation of Package.
+type DefaultPackage struct {
+	PackageName string
+	PackagePath string
+	// emitted in every file.
+	HeaderText []byte
+
+	// emitted only for a "doc.go" file.
```

Comment:
OK - it's obvious from the implementation, but can you please clarify it here too?